### ISSUE TITLE
Eyebrows not moving with brow morph animations

### ISSUE THREAD

**Issue by Colorfingers** [vircadia/vircadia#1538]
I have the following morphs for the brow on my avatar: 

BrowsU_R
BrowsU_L
BrowsD_R
BrowsD_L
BrowsU_C

I have the same morphs on the eyebrow attachments... The eyebrows are also weighted to the head.  I noticed in Vircadia that the eyebrows were not morphing to the same animation as the brows.  It should work the same as the mouth, teeth and tongue animations but it does not.

**Comment by ctrlaltdavid** [comment]
@Colorfingers Which version of Interface - current 2022.1.0 release or own build from latest source?

**Comment by Colorfingers** [comment]
I should try it on the current... was on last when I first noticed it... just getting around to investigating it.

**Comment by Colorfingers** [comment]
OK... trying on 2022.1.0 and it still does not seem to be working.  If you watch the brow with some shadow on it you will notice it is moving but the eyebrow does not.  Eyelashes are working as expected. Here is the avatar I'm experimenting with: https://hapticmonkey.s3.us-west-1.amazonaws.com/avatar.glb

**Comment by Aitolda** [comment]
Hey there. I just brought that avatar in and using selfie mode I've been watching it with the default animations, talking, etc and looking for misalignments and I'm not seeing any.  Can you tell me more about the context of when you are seeing this?

**Comment by Colorfingers** [comment]
Yes... sorry for the late reply.  The animation makes the forehead move up... If this isn't a morph I don't know what it is.. but the brows have the morphs and they don't move with it...
